lib/ClangImporter/ImportType.cpp

```diff
@@ -2203,6 +2203,47 @@ static Type decomposeCompletionHandlerType(
   }
 }
 
+ImportedType ClangImporter::Implementation::importPropertyType(
```

Comment:
The name `importPropertyType` sounds like it's for any property type, but this is specifically for effectful properties. (Or even just async properties). Can you add `async` or `effectful` to the name?